Library/Services/INfieldSurveySampleService.cs

```diff
@@ -108,5 +109,13 @@ public interface INfieldSurveySampleService
         /// <param name="columnsToUpdate">The name and values of the columns to be updated</param>
         /// <returns>True if success, false otherwise</returns>
         Task<bool> UpdateAsync(string surveyId, int sampleRecordId, IEnumerable<SampleColumnUpdate> columnsToUpdate);
+
+        /// <summary>
+        /// Creates custom columns for the specified record in the survey
```

Comment:
not only custom columns
```suggestion
        /// Creates a single sample record with the specified columns
```